src/components/capture/GenerateButton.tsx

```diff
@@ -26,14 +25,11 @@ interface Props {
 function CaptureGenerateButton({
     entityType,
     disabled,
-    postGenerateMutate,
     createWorkflowMetadata,
 }: Props) {
     const rediscoveryRequired = useResourceConfig_rediscoveryRequired();
-
     const { generateCatalog, isSaving, formActive } = useDiscoverCapture(
         entityType,
-        postGenerateMutate,
```

Comment:
Now that this function is in a context the hook can fetch it manually and no longer needs passed.